src/visualizer/visualization.py

```diff
@@ -157,7 +188,7 @@ def visualize(self):
         frame_interval = 0.025
         animated = animation.FuncAnimation(self.figure,  # input a figure for animation
                                          self.__animate,  # input method to update figure for each frame
-                                         np.arange(len(self.data)-1),  # Enter a list for the previous method for each frame
+                                         np.arange(len(self.data)-2),  # Enter a list for the previous method for each frame
```

Comment:
Could you put some comments as to why you are subtracting by 2?

Reply:
because the output_vis.json file at least has 3 keys(moment , shapes and owner's information) , the method just needs moment . when I added owners in file , must I changed 1. also I'll comment about that.

Reply:
Thanks.